docs/asciidoc/static/include/pluginbody.asciidoc

```diff
@@ -0,0 +1,1021 @@
+:branch:          1.5
+:ls_version:		  1.5.0.beta1
+
+= How to write your own Logstash {plugintype} plugin
+
```

Comment:
I think it's better to dive into the info instead of leading with "meta" info. And while I'm slowly warming up to the closed up style for most terms (such as plugin), HOWTO kinda makes me cringe. :-) I'd rework this as:

To develop a new {plugintype} for Logstash, you build a self-contained Ruby gem that
lives in its own GitHub repository. You can use the example {plugintype} implementation
as a starting point. (If you're unfamiliar with Ruby, you can find an excellent 
quickstart guide at {url}.) 

Note: As of Logstash 1.5, all plugins are self-contained Ruby gems. This change makes it
possible to develop and release plugins separately. In previous versions,
plugins were part of the core Logstash distribution.